cmd/conveyor/run_cmd_test.go

```
@@ -95,13 +97,15 @@ func runTaskScenario(t *testing.T, input string, auto, terminal bool) (taskRunSt
 	}}
 	sessions := map[string]string{}
 	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.Header.Get("Authorization") != "Bearer user-credential" {
+		authorization := r.Header.Get("Authorization")
+		if authorization != "Bearer user-credential" && !(r.URL.Path == "/mcp" && authorization == "Bearer child-agent-credential") {
```

golangci-lint (2.12.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)